### PR TITLE
Added forking instructions, fixed formatting

### PR DESCRIPTION
**Pull request by DavidLu1997**
Assuming anyone with direct commit access will not realistically need this guide.

@hmhealey please review :)

### CODE REVIEW

**Comment by it33** [comment]
+1 from IT, please pick a dev to review?

File: source/developer/developer-setup.md [hunk 1]
```diff
@@ -4,44 +4,46 @@ Developer Machine Setup
 ### Mac OS X ###
 
 1. Download and set up Docker Toolbox
-	1. Follow the instructions at http://docs.docker.com/installation/mac/
```
Comment (hmhealey) [inline source/developer/developer-setup.md]: You shouldn't need the backquotes around the URL here. Just having `[http://docs.docker.com/installation/mac/](http://docs.docker.com/installation/mac/)` should be fine. Same with the other URLs

File: source/developer/developer-setup.md [hunk 2]
```diff
@@ -4,44 +4,46 @@ Developer Machine Setup
...
+	1. `cd ~/go`  
+	2. `mkdir -p src/github.com/<username>`  
+	3. `cd src/github.com/<username>`  
+	4. `git clone https://github.com/<username>/platform.git`  
+	5. `cd platform`
 
 5. Install godep with `go get github.com/tools/godep`
-5. Run unit tests on Mattermost using `make test` to make sure the installation was successful
-6. If the tests passed, you need to create a team and an admin account. You can choose a team name, email and password:
+6. Run unit tests on Mattermost using `make test` to make sure the installation was successful
```
Comment (hmhealey) [inline source/developer/developer-setup.md]: You don't need to change it back, but just an FYI that Markdown automatically numbers lists like this. It's kind of annoying in a lot of cases, but it makes it easier to have long lists like this

**Comment by hmhealey** [comment]
Just the one small thing with the links, otherwise it looks good

**Comment by DavidLu1997** [comment]
Everything's fixed :)

**Pull request merged**